main-support.ts

```diff
@@ -490,7 +498,24 @@ function extractMetadataForThisONEFile(
       imageElement.width = origWidth;
       imageElement.screens = computeNumberOfScreenshots(screenshotSettings, duration);
 
-      extractMetaCallback(imageElement);
+      // !!!! WARNING !!!!!! NOT A PURE FUNCTION ANY MORE !!!!!!!!!!!!!!!
+      if (codeRunningOnMac) {
+        tag.getTags(filePath, (tagErr: any, tags: string[]) => {
+          if (tagErr) {
+            console.log('tags error!!!');
+            extractMetaCallback(imageElement);
+          }
+          console.log('tags found:');
+          console.log(tags);
+          if (tags.length) {
+            imageElement.tags = tags;
```

Comment:
⚠️⚠️⚠️ I think this will overwrite any user entered tags if you move the file within the video folder! ⚠️⚠️⚠️

Reply:
Great catch! 👀 

This code is WIP placeholder just to test out the library.

Thank you for adding the note -- I'll definitely not forget to fix it before merging 🤝

Reply:
I'm planning on adding my `.tag` file support after this, so we can probably make a nice way to implement both! 👍